docs/examples/frameworks/tensorflow/idx_files/train.idx

```diff
@@ -0,0 +1,47 @@
+0 171504
```

Comment:
is this file meant to be part of the PR?

Reply:
Leftover. Fixed